tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -1,10 +1,12 @@
 from pathlib import Path
+from unittest.mock import patch
 
 import pytest
 
 from tests.llmobs._utils import _expected_llmobs_llm_span_event
 
 from .test_anthropic import ANTHROPIC_VERSION
+from .utils import MOCK_MESSAGES_CREATE_REQUEST
 from .utils import tools
```

Comment:
I know you're just following the pattern here but let's avoid relative imports and convert these to absolute imports